### DISCUSSION TITLE
[Feature] Favourite and Add Pictures from shared album

### DISCUSSION THREAD

**Discussion by cclarke591** [immich-app/immich#14434] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My partner created an album for us to share pictures that we have taken of each other. I would love to be able to favourite the photos she adds, as well as add those photos to my own library.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#12614
